### DISCUSSION TITLE
How can I change the minimum value for the "Y" in a Line Series

### DISCUSSION THREAD

**Discussion by vciioffi** [patrykandpatrick/vico#715] [Questions] [answered]
Having a list like the next one: [79638116, 80415159, 80684694, 80914547]
I would like to set the min value with the lowest number from my list, instead of having a value of 0 like in the chart of the following image. Thanks in advance
![imagen](https://github.com/patrykandpatrick/vico/assets/99197513/7aaaf50a-f14f-4151-96e5-1519352c124b)

**Comment by patrickmichalik** [comment, marked as answer]
Hello! You can do this via `AxisValueOverrider`. Please see [here](https://patrykandpatrick.com/vico/wiki/cartesian-charts/layers/#axisvalueoverrider).

**Reply by vciioffi** [reply]
Thank you so much!!

**Reply by patrickmichalik** [reply]
You’re welcome!
